src/main/java/seedu/address/logic/commands/schedule/AddEventCommand.java

```diff
@@ -33,13 +34,15 @@
             + PREFIX_EVENT_DESCRIPTION + "EVENT_DESCRIPTION "
             + PREFIX_DATE + "DATE "
             + "[" + PREFIX_TIME + "TIME] "
-            + "[" + PREFIX_DURATION + "DURATION]"
+            + "[" + PREFIX_DURATION + "DURATION] "
+            + "[" + PREFIX_RECUR_FREQUENCY + "RECUR_FREQUENCY] "
             + "\nExample: " + COMMAND_WORD + " "
```

Comment:
```suggestion
            + "[" + PREFIX_RECUR_FREQUENCY + "RECUR_FREQUENCY] \n"
            + "Example: " + COMMAND_WORD + " "
```
Instead of adding new line to the start of a line. I think the convention is to add to the end of the previous line.